andela/vale-ah-backend
# #164139781 Tag User Recipes

#### What does this PR do?
- Creates an endpoint that allows users to tag their recipes
#### Description of Task to be completed?
- write tests for the endpoint
- create a migration to add `tags` column to the `recipes` table
- update the `Recipe` model to include the `tags` property
- create a `PUT /api/recipes/:slug/tags` route
- create a controller for the endpoint
- write an `isRecipeOwner` middleware function
- document the endpoint with `Swagger`
#### How should this be manually tested?
- ensure that the build is passing on `Travis`
- pull the branch and test locally using `Postman`
#### Any background context you want to provide?
Tagging refers to the metadata that is assigned to a piece of content by the content creator and the readers/users of the content. Tagging allows for organization and or categorization of content
#### What are the relevant pivotal tracker stories?
[#164139781](https://www.pivotaltracker.com/story/show/164139781)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
